web-marketplace/src/components/organisms/Order/Order.Summary.tsx

```diff
@@ -0,0 +1,189 @@
+import { msg, Trans } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+
+import { DenomIconWithCurrency } from 'web-components/src/components/DenomIconWithCurrency/DenomIconWithCurrency';
+import { BlockchainIcon } from 'web-components/src/components/icons/BlockchainIcon';
+import CertifiedDocumentIcon from 'web-components/src/components/icons/CertifiedDocumentIcon';
+import CreditsIcon from 'web-components/src/components/icons/CreditsIcon';
+import { PaymentInfoIcon } from 'web-components/src/components/icons/PaymentInfoIcon';
+import { SupCurrencyAndAmount } from 'web-components/src/components/SupCurrencyAndAmount/SupCurrencyAndAmount';
+import QuestionMarkTooltip from 'web-components/src/components/tooltip/QuestionMarkTooltip';
+
+import { Link } from 'components/atoms';
+
+import { OrderSummarySection } from './Order.Summary.Section';
+import { OrderSummaryRow } from './Order.SummaryRow';
+import {
+  BlockchainDetailsData,
+  CreditsData,
+  OrderSummarySectionProps,
+  PaymentInfoData,
+  RetirementInfoData,
+} from './Order.types';
+
+interface OrderSummaryProps {
+  retirementInfo: OrderSummarySectionProps;
+  blockchainDetails: OrderSummarySectionProps;
+  creditsData: OrderSummarySectionProps;
+  paymentInfo: OrderSummarySectionProps;
```

Comment:
why setting them all as `OrderSummarySectionProps`?

Reply:
Since the update of the `title` and `icon` as props this is doesn't need to be like this, thanks